frontend/src/components/login/LoginForm.tsx

```diff
@@ -23,28 +23,29 @@ export const LoginForm: React.FunctionComponent = () => {
       {error && <p>{error}</p>}
       <h1>Log In</h1>
       <form onSubmit={tryLogin}>
-        <label className="field">
-          Username
+        <div className="form-floating mb-3">
           <input
-            type={"text"}
-            value={username}
+            type="text"
+            className="form-control"
+            id="username"
             placeholder="Username"
-            required
             onChange={(event) => setUsername(event.target.value)}
           />
-        </label>
-        <label className="field">
-          Password
+          <label htmlFor="username">Username</label>
+        </div>
+        <div className="form-floating mb-3">
           <input
-            type={"password"}
-            value={password}
+            type="password"
+            className="form-control"
+            id="password"
             placeholder="Password"
-            required
```

Comment:
Shouldn't the `value` still be set to `password`, and this input still be `required`? Don't just change this blindly - I'm not sure which is correct, and if you've changed this for a reason I'm sure you're right to!